packages/pieces/localai/src/index.ts

```diff
@@ -0,0 +1,32 @@
+import { PieceAuth,Property, createPiece } from '@activepieces/pieces-framework';
+import { askLocalAI } from './lib/actions/send-prompt';
+
+export const localaiAuth = PieceAuth.CustomAuth({
+  
+  description: `
+  Use LocalAI to generate text.
+  `,
+  props: {
+      base_url: Property.ShortText({
+          displayName: "Server URL",
+          description: "LocalAI Instance URL",
+          required: true,
+      }),
+      access_token: PieceAuth.SecretText({
+          displayName: "Access Token",
+          description: "LocalAI Access Token",
+          required: false,
+      })
+  },
+  required: true,
+})
+export const openai = createPiece({
+  displayName: 'LocalAI',
+  description: 'Use LocalAi to generate text',
+  minimumSupportedRelease: '0.5.0',
+  logoUrl: 'https://avatars.githubusercontent.com/u/128192713?s=48&v=4',
```

Comment:
Can you please edit to `https://cdn.activepieces.com/pieces/localai.jpeg`

The pr doesn't allow maintainers to edit

Reply:
done